particles/Blackjack/BUILD

```diff
@@ -22,6 +22,5 @@ arcs_kt_particles(
         "Player.kt",
     ],
     package = "arcs.tutorials.blackjack",
-    platforms = ["wasm"],
```

Comment:
Do we need to update any SDK docs? Are there ramifications here, like, do we need to file a follow-up issue for ourselves to see about hiding the needed JVM interface-class/schema-gen build within our build macros?